src/main/java/dk/sdu/seb05/semesterprojekt/PresentationLayer/CreateCreditsController.java

```diff
@@ -41,11 +41,11 @@
 
     public void initialize() {
         fulcrum = PresentationSingleton.getInstance();
-        FunctionType[] funcs = FunctionType.values();
-        for (int i = 0; i < funcs.length; i++) {
-
-            functionTypeExistChoiceBox.getItems().add(funcs[i].toString());
-            functionTypeNewChoiceBox.getItems().add(funcs[i].toString());
+        fulcrum.changeView("Tilføj credits");
```

Comment:
Hvad sker der her? Det her virker vel ikke? For er `changeView` ikke den der læser en fil. Der er vel ingen fil der hedder "Tilføj credits"? Eller misforstår jeg?

Reply:
Ups, det har du ret i 😅